Rules/CTF/mapcycle.cfg

```
@@ -16,11 +16,14 @@ mapcycle =
 			Maps/CTF/bunnie_Luthadel.png;
 				#Ej
 			Maps/CTF/Ej_Cataracts.png;
+			Maps/CTF/Ej_Chambers.png;
 				#Ferrezinhre
 			Maps/CTF/Ferrezinhre_Highlands.png;
 				#Geti
 			Maps/CTF/HearthPlains.png;
 			Maps/CTF/Snatchmark.png;
+				#ffsff
+			Maps/CTF/PlatformFlats.png;
```

```suggestion
			Maps/CTF/ffsff_PlatformFlats.png;
```

See if you can accept that suggestion from phone. If not I'll do it manually after merging.

---

Accepted, but I don’t see a commit anywhere, not sure.